skills/mobile-use-setup/references/platform-vs-local.md

```
@@ -0,0 +1,138 @@
+- Optimized model selection handled for you
+- Task management UI
+- Built-in observability and logging
+
+### Cons
+- Requires internet connection
+- Dependent on Minitap service
+- Less control over model selection
+
+### Setup
+
+1. Sign up at https://platform.minitap.ai
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)